src/porepy/numerics/time_step_control.py

```diff
@@ -137,10 +198,14 @@ def __init__(
         # Minimum and maximum allowable time steps
         self.dt_min, self.dt_max = dt_min_max
 
-        # Maximum amount of iterations
+        # Maximum number of iterations
+        # TODO: This is really a property of the nonlinear solver. A full integration will
```

Comment:
Agree. This is a natural next step, after we have integrated this into the models.